### PR TITLE
[PP][OPS-7764] readme update

### PR DESCRIPTION
**Pull request by pawelpanasewicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
 
-The stub must be run on your local machine. You can do this via service manager.
+# Websockets and MDTP
 
-Warning: You will also need to run stubs locally if you want to use a stubbed test environment. This is due to the frontend service using websockets - which are not supported on the MDTP platform.
+Unfortunatelu because this service exposes a websocket endpoint it can't be deployd to MDTP platform. MDTP doesn't support such technology.
+Therefore this service has to always run on developer's machine.
 
-#### Service manager
+It has to also run locally when testing F2F on integrated environment (like Development, QA or Staging).
```
Comment (gitwojciech) [inline README.md:11]: QA is configured to work with PED. Stub can olny be used for Dev and Local.

Comment (gitwojciech) [reply README.md:11]: Could you please mention that ENV needs to be configured to work with Stub or PED. At the momemnt QA is configured to work with PED

File: README.md [hunk 2]
```diff
@@ -1,33 +1,48 @@
 
-# smartpay-connect-stub
-This stub supports the Face to Face service, and is designed to behave like a PED (Pin enabled device). The PED model that is being used on the service is the Lane 3000.  
+# About
 
-### How to run
+The smartpay-connect-stub is a stub microservice used in F2F project. It emulates group of Barclaycard related systems involved in taking a card payment. Those are a _smartpay-connect-proxy_ and _smartpay-connect_ connected to the PED (Pin Enabled Device). This service emulates the user interaction with PED. Various scenarios can be choosen definint user behaveiour for successfull and faile payments.
```
Comment (gitwojciech) [inline README.md:4]: You can reuse your diagrams from proxy and replace proxy and smartpay with Stub

File: README.md (hunk 1 above)
Comment (pawelpanasewicz) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (pawelpanasewicz) [reply README.md:4]: +1

**Pull request merged**